src/darwin/frida-helper-backend-glue.m

```diff
@@ -1450,6 +1466,74 @@ static void frida_darwin_helper_backend_launch_using_sbs (NSString * identifier,
   return killed_pid;
 }
 
+static gboolean
+frida_has_active_prewarm (gint pid)
+{
+  gboolean result = false;
+  int mib_argmax[] = { CTL_KERN, KERN_ARGMAX };
+  int mib_args[] = { CTL_KERN, KERN_PROCARGS2, 0 };
+  size_t size;
+  gint32 arg_max = 0, argc;
```

Comment:
Think it would be clearer to initialize this explicitly on the line before it's used, i.e. just before the `sysctl()`.